src/Translator.php

```diff
@@ -73,11 +77,9 @@ public function __construct()
                 "available" => ['en', 'nl', 'fr', 'de'],
             ]
         );
-    }
 
-    public function setLanguage($langCode): void
```

Comment:
Is there a technical reason that we have to remove this right now? Could we deprecate it using `@deprecate` instead and remove it in a later v2 instead of releasing a v2 now with just this one breaking change?

I think ideally we'd group breaking changes for major releases for packages like this, intended to be used by 3rd parties, every once in a while and not on an ad-hoc basis per PR. I know we did a lot of breaking changes earlier but that was intended so we could have a v1 that didn't have to become v2, v3, etc in a matter of days.

I do agree that this method is now pretty useless though.

Alternatively we can merge this with the breaking change but not tag a v2 yet, and depend on `dev-main` in udb3 so we already have these optimizations. But I'm not sure if that's any better. 🤔

Reply:
As far as I know we're the only app consuming the latest (already broken) version, so we might get away with it. If not, it's easy to revert this BC break and deprecate it as you suggest. Would avoid depending on `dev-main` where possible.

Reply:
Yes, we will definitely get away with it 😄 We'll do a `v2` release now* like you suggested, I just think we should start grouping (breaking) changes and planning our releases a bit more in the future, instead of always going with "1 PR with coincidentally 1+ breaking changes = immediately 1 new major version". Because then we end up with `v25` pretty soon. Maybe not in this library per se, but I think it would just be a good habit to start practicing so we don't get an overgrowth of versions in the packages that we will keep after internalizing and archiving most.

*Maybe not _now_, but once III-3556 is 100% done and we want to pull that into udb3-silex? In case any other breaking changes pop up?

Reply:
I agree, let's finish everything before tagging :D